js/messages.js

```diff
@@ -1,114 +1,133 @@
+      roleColumn.append(creditListingWrapper);
     });
+  }
+}
 
-    for (var i = 0; i < roleElements.length; i++) {
-        let roleColumn = roleElements[i];
-        let staffList = roleMapping[i];
+function renderTwitterArtCredits(twitter_art_data) {
+  let creditsContainer = document.getElementById("credits");
+  // Create new "section"
+  let creditsColumn = document.createElement("section");
+  creditsColumn.classList.add("credits-column");
+  creditsColumn.id = "twitter-images";
+  let twitterImagesHeader = document.createElement("h2");
+  twitterImagesHeader.textContent = "Twitter Images";
+  creditsColumn.append(twitterImagesHeader);
+  const artList = document.createElement("ul");
 
-        staffList.forEach((credit) => {
-            let creditListing = document.createElement("li");    
-            let creditName = document.createElement("h2");
-            let creditRole = document.createElement("p");
-            creditName.textContent = credit["Staff name"];
-            creditListing.append(creditName);
-            if (credit["Twitter"]) {
-                let creditTwitterHeader = document.createElement("h3");
-                let creditTwitter = document.createElement("a");
-                creditTwitter.href = `https://twitter.com/${credit["Twitter"]}`;
-                creditTwitter.textContent = credit["Twitter"];
-                creditTwitterHeader.append(creditTwitter);
-                creditListing.append(creditTwitterHeader);
-            }
-            creditRole.textContent = credit["Role"];
-            creditListing.append(creditRole);
-            roleColumn.append(creditListing);
-        });
+  twitter_art_data.forEach((artData) => {
+    const artContainer = document.createElement("li");
+    const img = document.createElement("img");
+    img.src = `../images/Twitter Images/${artData["Artwork"]}`;
+    img.alt = artData["Alt"];
+    img.className = "credits-page-artwork";
+    artContainer.append(img);
+    if (artData["Twitter"]) {
+      let creditTwitter = document.createElement("a");
+      creditTwitter.href = `https://twitter.com/${artData["Twitter"]}`;
+      creditTwitter.textContent = artData["Staff name"];
+      artContainer.append(creditTwitter);
     }
+    artList.append(artContainer);
+  });
+  creditsColumn.append(artList);
+  creditsContainer.append(creditsColumn);
 }
 
 async function main() {
-    getCSVData(CURRENT_PAGE, async (csv_data) => {
-        switch (CURRENT_PAGE) {
-        case "/contributor-board.html":
-            renderMessages(csv_data);
-            break;
-        case "/credits.html":
-            renderCredits(csv_data);
-            break;
-        }
-    });
+  switch (CURRENT_PAGE) {
+    case "/contributor-board.html":
+      getCSVData(MESSAGES_CSV, renderMessages);
+      break;
+    case "/credits.html":
+      getCSVData(CREDITS_CSV, renderCredits);
+      getCSVData(TWITTER_ART_CREDITS_CSV, renderTwitterArtCredits);
```

Comment:
main change: reorganize code so that credits.html gets 2 csv's
note: these CSV's may not load in the same order each time. In an earlier iteration, this caused the credits sections to render in different orders when you refreshed the page lol. Now that I'm using CSS grid to explicitly specify the position each section should render in when it's ready, the order that these load in no longer seems to be an issue